src/notesubgroup.ts

```diff
@@ -12,14 +12,18 @@ import { Flow } from './tables';
 import { Modifier } from './modifier';
 import { Formatter } from './formatter';
 import { Voice } from './voice';
+import { ModifierContextState } from './modifiercontext';
+import { Note } from './note';
+import { Tickable } from './tickable';
```

Comment:
You can delete this. `Tickable` is not used.

Reply:
thanks! vscode must have inserted it when I was trying to get at one of the protected variables from Note / Tickable.